virtual_rainforest/entry_points.py

```diff
@@ -67,31 +69,70 @@ def _parse_command_line_params(
         raise to_raise
 
 
-def vr_run_cli() -> None:
+def vr_run_cli(args_list: Optional[list[str]] = None) -> None:
     """Configure and run a Virtual Rainforest simulation.
 
-    This program sets up and runs a simulation of the Virtual Rainforest model. At
-    present this is incomplete. At the moment a set of configuration files are read in
-    based on user supplied paths, these are converted to a single configuration file,
-    which is then output for further reference. This combined configuration is then used
-    to initialise a set of models.
-
-    The command accepts one or more paths to config files or folders containing config
-    files (cfg_paths). The set of config files found in those locations are then
-    combined and validated to make sure that they contain a complete and consistent
-    configuration for a virtual_rainforest simulation. The resolved complete
-    configuration will then be written to a single consolidated config file in the
-    output path with a default name of `vr_full_model_configuration.toml`. This can be
-    disabled by setting the `core.data_output_options.save_merged_config` option to
-    false.
+    This program sets up and runs a Virtual Rainforest simulation. The program expects
+    to be provided with paths to TOML formatted configuration files for the simulation.
+    The configuration is modular: a directory path can be used to add all TOML
+    configuration files in the directory, or individual file paths can be used to select
+    specific combinations of configuration files. These are combined and validated and
+    then used to initialise and run the model.
+
+    As an alternative to providing configuration paths, the `--install_example` option
+    allows users to provide a location where a simple example set of datasets and
+    configuration files provided with the Virtual Rainforest package can be installed.
+    This option will create a `vr_example` directory in the location, and users can
+    examine the input files and run the simulation from that directory:
+
+    `vr_run /provided/install/path/vr_example`
+
+    The output directory for simulation results is typically set in the configuration
+    files, but can be overwritten using the `--outpath` option. A log file path can be
+    provided for logging output - if this is not provided the log will be written to the
+    console.
+
+    The resolved complete configuration will then be written to a single consolidated
+    config file in the output path with a default name of
+    `vr_full_model_configuration.toml`. This can be disabled by setting the
+    `core.data_output_options.save_merged_config` option to false.
+
+    Args:
+        args_list: This is a developer and testing facing argument that is used to
+            simulate command line arguments, allowing this function to be called
+            directly.
```

Comment:
Maybe provide an example of how this should be used. For example, if I want to request to install the example this way, what should this list be? In other words, what's the equivalent to `--install_example /path/to/copy/the/example/to`, for example?